src/schemas/comment.ts

```diff
@@ -0,0 +1,7 @@
+export interface CommentPayload {
```

Comment:
I'd like to consider reusing the Message payload first.

`replyId` is like we are quoting another message?

Discussion is welcome.

Reply:
Agree, show my draft design here:

- add `MessageType.Comment`

- add `quoteId` to `MessagePayloadBase`

- methods in `Message` class
  - commentContent() -> comment() // differ with message.say()
  - commentRevoke() -> delete() // differ with message.recall()
  - commentReply() -> reply()